### PR TITLE
[now-next] Use patched version of yazl and trace API pages separate

### PR DESCRIPTION
**Pull request by ijjk**
This adds usage of a modified version of `yazl` that allows for adding pre-deflated buffers. It also adds tracing of API pages separately. These changes together reduce lambda creation times from `36s` to `7s` on front and reduce the lambda zip sizes from `9MB` to `2.7MB`

### CODE REVIEW

File: packages/now-next/package.json [hunk 1]
```diff
@@ -20,17 +20,16 @@
...
     "@zeit/node-file-trace": "0.2.13",
     "async-sema": "3.0.1",
     "fs-extra": "7.0.0",
     "get-port": "5.0.0",
-    "jszip": "3.2.2",
     "resolve-from": "5.0.0",
     "semver": "6.1.1",
-    "typescript": "3.5.2"
+    "typescript": "3.5.2",
+    "yazl": "https://github.com/ijjk/yazl"
```
Comment (Timer) [inline packages/now-next/package.json]: Please pin this to a specific SHA or tag.

File: packages/now-next/src/utils.ts [hunk 2]
```diff
@@ -1,33 +1,12 @@
...
-const compressBuffer = (buf: Buffer): Promise<Buffer> => {
-  return new Promise((resolve, reject) => {
-    zlib.deflateRaw(buf, (err, compBuf) => {
-      if (err) return reject(err)
-      resolve(compBuf)
-    })
-  })
-}
+const { ZipFile } = require('yazl')
+const crc32 = require('buffer-crc32')
```
Comment (Timer) [inline packages/now-next/src/utils.ts]: Why aren't these ES6 imports?

**Review by Timer — CHANGES_REQUESTED** [review]

File: packages/now-next/src/utils.ts (hunk 2 above)
Comment (ijjk) [reply packages/now-next/src/utils.ts]: Updated

File: packages/now-next/package.json [hunk 3]
```diff
@@ -19,18 +19,20 @@
...
     "async-sema": "3.0.1",
+    "buffer-crc32": "0.2.13",
     "fs-extra": "7.0.0",
     "get-port": "5.0.0",
-    "jszip": "3.2.2",
     "resolve-from": "5.0.0",
     "semver": "6.1.1",
-    "typescript": "3.5.2"
+    "typescript": "3.5.2",
+    "yazl": "https://github.com/ijjk/yazl#70949c55b482647669ce37023017b1514c42b33c"
```
Comment (TooTallNate) [inline packages/now-next/package.json:36]: Is there an upstream PR for the relevant change in your fork? If not, perhaps we should publish under `@zeit` scope on the registry.

Comment (Timer) [reply packages/now-next/package.json:36]: We're sending a PR, no reason to fork in the intermediary time IMO.

**Review by Timer — APPROVED** [review]
Great work.

File: packages/now-next/package.json (hunk 3 above)
Comment (ijjk) [reply packages/now-next/package.json:36]: PR here https://github.com/thejoshwolfe/yazl/pull/55

**Pull request merged**